src/apis/sourceimages.js

```diff
@@ -185,13 +186,18 @@ export default (state) => {
             formData[o + '[0]'] = typeof data === 'string' ? data : JSON.stringify(data)
           })
         }
+        if (options !== null) {
```

Comment:
unless explicitely passed as `null`, this probably will never happen. Did you want to set the default of `options` to `null` instead of `{}`?

Reply:
true, just removed the if